bower.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "assets",
-  "version": "0.0.0",
+  "name": "origin-web-console",
+  "version": "1.3.0",
```

Comment:
technically we'd be 1.3.0-alpha or whatever that format is since we haven't actually cut a release of 1.3.0